src/main/java/frc/robot/subsystems/elevator/ElevatorIONeo.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems.elevator;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.Encoder;
+
+public class ElevatorIONeo implements ElevatorIO {
+  private double m_wantedHeight;
+  private final CANSparkMax m_leaderMotor;
+  private final CANSparkMax m_followerMotor;
+  private final Encoder m_leaderEncoder;
+
+  private final PIDController m_elevatorPIDController;
```

Comment:
The PID should be in the subsystem class instead. See https://github.com/Mechanical-Advantage/RobotCode2023/blob/main/src/main/java/org/littletonrobotics/frc2023/subsystems/arm/Arm.java